ncm-metaconfig/src/main/pan/components/metaconfig/schema.pan

```diff
@@ -15,7 +15,6 @@ type ${project.artifactId}_config =  {
      'mode' : long = 0644
      'owner' : string = 'root'
      'group' : string = 'root'
-     'daemon' ? string[]
```

Comment:
ok, but afaik MS were also using metaconfig already so they probably also have an opinion
@ned21 any takes how to deprecate this?